dicomsort.py

```diff
@@ -189,10 +189,16 @@ def renameFiles(self):
             print("Preparing the list of files ...")
 
         allFiles = []
-        for root, subFolders, files in os.walk(self.options['sourceDir']):
-            for file in files:
-                file = os.path.join(root,file)
-                allFiles.append(file)
+        if self.options['sourceDir'] == "":
+            for line in sys.stdin:
+                line = line.strip()
+                if os.path.isfile(line):
+                    allFiles.append(line.strip())
+        else:
+            for root, subFolders, files in os.walk(self.options['sourceDir']):
```

Comment:
I think it would be cleaner if the filelist handling were factored out, so that the `allFiles` variable was populated by a method that handles the implementation.  I'm not sure it's required, but someday we might want to be able to have the filelist from stdin include directories and not just files, so the code for walking through the directory tree could be re-used to handle a directory path that come either from the command line, or from a file, or from stdin.